config/manager/manager.yaml

```diff
@@ -68,6 +68,10 @@ spec:
         imagePullPolicy: IfNotPresent
         name: manager
         env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
```

Comment:
This won't be necessary anymore. See my comment https://github.com/cybozu-go/fin/pull/145/files#r2443660474.